lib/menu.rb

```diff
@@ -0,0 +1,71 @@
+require 'json'
+
+class Menu
+
+     attr_accessor :customer_bill, :customer_order, :taxes, :sum, :amount
+     attr_reader :menu
+
+  def initialize
+       file = open("hipstercoffee.json")
+       json = file.read
+       @menu = JSON.parse(json)
+       @customer_order = []
+       @customer_bill = []
+       @total = 0
+       @taxes = 0
+       @quantity = 1
+
+  end
+
+  def order(booking, quantity)
+       @menu[0]["prices"][0].select do |key,value|
+          if booking == key and booking.include?'Muffin'
+       @customer_order.push key => value * quantity * 0.9
+     elsif booking == key and !booking.include?'Muffin'
```

Comment:
Lint/SpaceBeforeFirstArg: Put space between the method name and the first argument.<br>Style/AndOr: Use `&&` instead of `and`. (https://github.com/bbatsov/ruby-style-guide#no-and-or-or)